src/nms-openwire.csproj

```diff
@@ -0,0 +1,44 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFramework>netstandard2.0</TargetFramework>
```

Comment:
lets make this multi-target see https://github.com/apache/activemq-nms-api/blob/master/src/nms-api/nms-api.csproj